Assets/scripts/ItemManager.cs

```diff
@@ -184,17 +211,37 @@ private void AddDemoDataToDB()
                                                         price,
                                                         sellerURL,
                                                         itemDesc,
-                                                        "",
-                                                        "",
-                                                        "",
-                                                        "",
+                                                        category,
+                                                        brand,
+                                                        designer,
+                                                        specs,
                                                         noClick);
                 dbCommand.CommandText = insertRecord;
                 dbCommand.ExecuteScalar();
             }
             dbCommand.Cancel();
         }
     }
-    //https://www.youtube.com/watch?v=5p2JlI7PV1w
-    //https://www.youtube.com/watch?v=AvuuX4qxC_0
+
+    public void updateNumberOfClicks(Item item)
+    {
+        using (dbconn = new SqliteConnection(dbURL))
+        {
+            dbconn.Open();
+            dbcmd = dbconn.CreateCommand();
+            sqlQuery = string.Format("UPDATE Item set noClick = @noClick where ID = @id ");
+
+            SqliteParameter P_update_name = new SqliteParameter("@noClick", item.getNumberOfClick());
+            SqliteParameter P_update_id = new SqliteParameter("@id", item.GetItemID());
+
+            dbcmd.Parameters.Add(P_update_name);
+            dbcmd.Parameters.Add(P_update_id);
+
+            dbcmd.CommandText = sqlQuery;
+            dbcmd.ExecuteScalar();
+            dbconn.Close();
```

Comment:
The dbconn will close by its self. I think we have to do call ```dbcmd.close();``` to close the command

Reply:
There is no dbcmd.close(), t